### ISSUE TITLE
iTerm2 color scheme based on DA UI

### ISSUE THREAD

**Issue by gurqn** [ihodev/sublime-da-ui#28]
Would it be possible creating the dark color scheme that is used in Sublime for the iTerm application on macOS?

**Comment by ihodev** [comment]
Yes, I have plans to port it to another tools. Just a matter of time :wink: I'll be back when it will be ready.

**Comment by gurqn** [comment]
Thanks a lot for quick response :-)

**Comment by schrodincat** [comment]
This would be awesome addition indeed and it should be relatively easy to do. I'd love my workspace to have coherent theme across apps.
